Uchu.World/Objects/GameObjects/GameObject.cs

```diff
@@ -148,12 +148,19 @@ protected GameObject()
                     this.GetComponent<LuaScriptComponent>() != default) return;
                 var scriptName = ((string) scriptNameValue).ToLower();
                 Logger.Debug($"{this} -> {scriptNameValue}");
+                var a = false;
```

Comment:
Can this variable name be changed to something more specific, like `scriptLoaded`?

Reply:
Will do.